base/float.jl

```diff
@@ -844,8 +844,10 @@ end
 """
     floatmin(T = Float64)
 
-Return the smallest positive normal number representable by the floating-point
-type `T`.
+Return the smallest positive non-subnormal number representable by the floating-point type T.
+This is not the negation of `floatmax`. See the Extended help section for more info.
```

Comment:
There should be a paragraph break here
```suggestion

This is not the negation of `floatmax`. See the Extended help section for more info.
```